subprojects/core/src/main/java/org/gradle/configuration/project/BuildScriptProcessor.java

```diff
@@ -15,28 +15,26 @@
  */
 package org.gradle.configuration.project;
 
+import org.gradle.api.internal.configuration.ScriptPluginApplicator;
 import org.gradle.api.internal.project.ProjectInternal;
-import org.gradle.configuration.ScriptPlugin;
-import org.gradle.configuration.ScriptPluginFactory;
 import org.gradle.internal.time.Timer;
 import org.gradle.internal.time.Timers;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class BuildScriptProcessor implements ProjectConfigureAction {
     private static final Logger LOGGER = LoggerFactory.getLogger(BuildScriptProcessor.class);
-    private final ScriptPluginFactory configurerFactory;
+    private final ScriptPluginApplicator scriptPluginApplicator;
 
-    public BuildScriptProcessor(ScriptPluginFactory configurerFactory) {
-        this.configurerFactory = configurerFactory;
+    public BuildScriptProcessor(ScriptPluginApplicator scriptPluginApplicator) {
+        this.scriptPluginApplicator = scriptPluginApplicator;
     }
 
-    public void execute(ProjectInternal project) {
+    public void execute(final ProjectInternal project) {
         LOGGER.info("Evaluating {} using {}.", project, project.getBuildScriptSource().getDisplayName());
```

Comment:
let's get rid of this logging, as wrapping script execution in a build operation will provide this kind of data anyway.